samples/dotnet/kernel-syntax-examples/Example22_b_OpenApiSkill_Jira.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.WebApi.Rest;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Authentication;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Skills;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using RepoUtils;
+
+public static class Example22_b_OpenApiSkill_Jira
+{
+    public static async Task RunAsync()
+    {
+        var kernel = new KernelBuilder().WithLogger(ConsoleLogger.Log).Build();
+        var contextVariables = new ContextVariables();
+
+        string s_serverURL = "https://<jiraProject>.atlassian.net/rest/api/latest/";
+        contextVariables.Set("server-url", s_serverURL);
+
+        IDictionary<string, ISKFunction> jiraSkills;
+        var tokenProvider = new BasicAuthenticationProvider(AuthenticateWithBasicApiTokenAsync);
+
+        // The bool bUseLocalFile can be used to toggle the ingestion method for the openapi schema between a file path and a URL
+        bool bUseLocalFile = true;
+        if (bUseLocalFile)
```

Comment:
The scenario we're trying to show with Jira is being able to pull a Power Platform Connector directly. Could we simplify this to just using the api definition URL and remove the checked-in openapi.json?

Reply:
I feel like its worth showing both approaches?
But the other concern with the full open api definition for jira is that not all the operations work. like createIssue doesnt follow the api schema and therefore doesnt work.

Reply:
@adrianwyatt, thoughts?

Reply:
Ah - okay. If we're modifying the openapi JSON, than checking it in and loading it from disk makes sense. Let's also put a README next to the jira openAPI json explaining why we have a separate copy and why.